tests/pages/components/conversations/conversation-thread.js

```diff
@@ -1,15 +1,15 @@
 import { collection } from 'ember-cli-page-object';
 import conversationComposer from 'code-corps-ember/tests/pages/components/conversations/conversation-composer';
-import conversationPart from 'code-corps-ember/tests/pages/components/conversations/conversation-part';
+import conversationPartComment from 'code-corps-ember/tests/pages/components/conversations/conversation-part-comment';
 
 export default {
   scope: '.conversation-thread',
   conversationComposer,
   conversationParts: collection({
     scope: '.conversation-thread__messages',
-    item: conversationPart,
+    item: conversationPartComment,
     // when a collection scope is specified, itemScope is required
     // in order to correctly detect when there is 0 items
-    itemScope: conversationPart.scope
+    itemScope: conversationPartComment.scope
   })
 };
```

Comment:
This is problematic. The conversation thread needs to have a single collection of items and all of them need to be used interchangeably. That means that, for all types of conversation parts, we'd need a single page object, which is sort of a combination of all of them.

I'm not really seeing a better way to do this.

That means, we

- remove `pages/components/conversations/conversation-part-comment`
- remove `pages/components/conversations/conversation-part-closed`
- add back `pages/components/conversations/conversation-part` which is a combination of both

```Javascript
import {
  attribute,
  hasClass,
  triggerable
} from 'ember-cli-page-object';

export default {
  scope: '.conversation-part',

  body: {
    scope: '[data-test-body]'
  },

  photo: {
    scope: '[data-test-target-photo]',
    url: attribute('src')
  },

  photoContainer: {
    scope: '[data-test-target-photo-container]',
    mouseenter: triggerable('mouseenter')
  },

  sentAt: {
    scope: '[data-test-sent-at]'
  },

  closedAt: {
    scope: '[data-test-closed-at]'
  },

  isComment: hasClass('conversation-part--comment'),
  isClosed: hasClass('conversation-part--closed'),
  isByCurrentUser: hasClass('conversation-part--is-self')
};
```

- then, we import that page object into this file and use it as `conversationPart` again. Basically, you can then undo the changes in this specific file.

A couple of tests may break due to renaming `closedByCurrentUser` and `sentByCurrentUser` into `isByCurrentUser`, but those ought to be easily fixed.